CommonConfig.mk

```diff
@@ -42,6 +42,7 @@ TARGET_KERNEL_ARCH := arm64
 TARGET_KERNEL_HEADER_ARCH := arm64
 TARGET_KERNEL_CROSS_COMPILE_PREFIX := aarch64-linux-android-
 BOARD_KERNEL_IMAGE_NAME := Image.gz-dtb
+TARGET_EXFAT_DRIVER := sdfat
```

Comment:
maybe move it above TARGET_FS_CONFIG_GEN?

Reply:
and what it change?

Reply:
it'll make me feel better, much better.

Reply:
:)